definitions/infra-mongodb_instance/tests/MetricRaw.json

```diff
@@ -0,0 +1,9 @@
+[
```

Comment:
The problem in the test is that you are saying that the entity will be named by the `instance` attribute. but that attribute doesn't exist here.

That will make the rule not match and will not create anything.

Reply:
Understood. Fixed 👍